wordle-web/src/views/HomeView.vue

```diff
@@ -1,9 +1,7 @@
 <template>
-  <main>
-    <TheWelcome />
-  </main>
+  <main></main>
```

Comment:
looks like this returns nothing

Reply:
I removed this file entirely.. we weren't using it.